### PR TITLE
Remove empty return to allow else clause in post-list shortcode.

### PR DESCRIPTION
**Pull request by devilgate**
### Pull Request Checklist

- [x] I’ve read the [guidelines for contributing](https://github.com/getnikola/nikola/blob/master/CONTRIBUTING.rst).
- [x] I updated AUTHORS.txt and CHANGES.txt (if the change is non-trivial) and documentation (if applicable).
- [x] I tested my changes.

### Description

Removes an empty return which prevented the template being called if no posts were returned, and hence stopped the template's `else` clause from executing.

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -4,6 +4,7 @@ New in v8.1.1
 Bugfixes
 --------
 
+* Allow else clause in post-list plugin. (Issue #3436)
```
Comment (Kwpolska) [inline CHANGES.txt:7]: This should be in the “new in master” block, please rebase on origin/master and fix the changelog.

Comment (devilgate) [reply CHANGES.txt:7]: Sorry, what's the “new in master” block? I can't find a section like that.

Comment (Kwpolska) [reply CHANGES.txt:7]: You’re looking at an outdated copy of this file, pull from upstream.

image

Comment (devilgate) [reply CHANGES.txt:12]: Ah! Got it. Thanks.

**Pull request merged**